test/index.js

```diff
@@ -23,6 +23,7 @@ var it = lab.it;
 
 
 var logPath = __dirname + '/test1.log';
+var flags = { flags: 'a' };
```

Comment:
@cjihrig @devinivy Thanks for the feedback. I just want to clarify the unit test comment as the others are straightforward updates. Are you asking to see each test executed multiple times with and without different options (`logPath` and `writeStreamOptions`)? Or, are you asking me to modify the current tests so that each one is running different registration options?